CHANGELOG/CHANGELOG-1.4.md

```diff
@@ -26,7 +26,7 @@
 - [Enhance robustness test to cover XFS](https://github.com/etcd-io/bbolt/pull/707)
 
 ### Other
-- [Bump go toolchain version to 1.22.2](https://github.com/etcd-io/bbolt/pull/712)
+- [Bump go toolchain version to 1.22.3](https://github.com/etcd-io/bbolt/pull/712)
```

Comment:
v1.4.0-alpha.1 has already been released. Please revert this change, and I will take care of it later. Thanks

Reply:
Done